src/components/SelectInput/index.tsx

```diff
@@ -0,0 +1,57 @@
+import classNames from 'classnames';
+import React, { FC } from 'react';
+
+import styles from './SelectInput.module.css';
+
+import { ISelectInputProps } from './ISelectInput.d';
+
+const SelectInput: FC<ISelectInputProps> = ({
+  id,
+  label,
+  placeholder,
+  options,
+  disabled = false,
+  error = false,
+  errorMessage = '',
+}) => {
+  const selectInputClasses = classNames(styles['select'], {
+    [styles['error']]: error,
```

Comment:
there's no need to have to props to check for errors, use only the `errorMessage` prop
```suggestion
    [styles['error']]: Boolean(errorMessage),
```

Reply:
What if I want an input with error state but with no error message?

Reply:
Better give the user more flexibility in case there is a use case for such a thing that we didn't think of, especially it's not that expensive to keep.